source/recurring_payments/index.html.md.erb

```diff
@@ -0,0 +1,391 @@
+
+1. Make your user aware of the terms, conditions, and purpose of the recurring payments. They must understand that you’ll use their payment details to take future payments.
+
+1. Record the user’s consent to your terms and conditions.
+
+1. Add `set_up_agreement: "{AGREEMENT_ID}"` to your request when creating the user’s first payment. Replace `{AGREEMENT_ID}` with the `agreement_id` the API sent when you created the agreement. You’ll also need to include the usual required parameters for creating a payment - `amount`, `description`, `reference`, and `return_url`.
+
+Your request to create this first payment should look something like this:
+
+```json
+{
+  "amount": 1000,
+  "description": "First payment for 2022/23 council tax.",
+  "return_url": "https://service.gov.uk",
+  "set_up_agreement": "cgc1ocvh0pt9fqs0ma67r42l58"
+}
+```
+
+`set_up_agreement` tells our API to save your user’s card details when they complete this payment. We use a token to represent the user's card details. This token is associated with the agreement with the ID you send.
+
+You can also use `set_up_agreement` to reactivate a cancelled or expired agreement.
+
+When your user is making their first payment, GOV.UK Pay will show your user the `description` you sent when you created the agreement.
+
+Once the user has made the payment, the agreement’s `status` will be `active`. You can use active agreements to take recurring payments.
+
+If the user’s payment fails, their payment details are not saved and the agreement’s `status` remains `created`.
+
+Read more about creating payments in [our guidance for taking payments](/making_payments) or our [API reference page for creating payments](/api_reference/create_a_payment_reference).
+
+## Taking recurring payments
+
+Once you’ve set up the agreement, you can use it to take recurring payments without the paying user interacting with your service.
+
+Your user must know when you are going to take payments. For example, if you take payments on an agreed schedule, you must make your user aware of which day of the month you'll take payments. If you take payments irregularly, you must let your user know you're going to take a payment.
+
+To take a recurring payment, create a payment through the API (`POST v1/payments`) and include the following parameters in the request body:
+
+* `"authorisation_mode" : "agreement"`
+* `"agreement_id": "{AGREEMENT_ID}"`
+
+Replace `{AGREEMENT_ID}` with the `agreement_id` you just set up.
+
+Your request should look something like this:
+
+```json
+{
+  "amount": 1000,
+  "description": "Council tax payment for 2022/23",
+  "authorisation_mode": "agreement",
+  "agreement_id": "cgc1ocvh0pt9fqs0ma67r42l58"
+}
+```
+
+The API returns a standard create payment response. Successful responses include the `agreement_id` you sent and `authorisation_mode` is `agreement`.
+
+To check if a payment authorised successfully, get information about it through the API (`GET v1/payments/{PAYMENT_ID}`).
```

Comment:
I think it might useful to point out that if they immediately call this after creating a payment, we might not know if it’s successful or not yet.